examples/utils/example-utils/src/modelLoader/modelLoader.ts

```diff
@@ -40,13 +53,14 @@ export class ModelLoader<ModelType> implements IModelLoader<ModelType> {
     private readonly generateCreateNewRequest: () => IRequest;
 
     // TODO: See if there's a nicer way to parameterize the createNew request.
+    // Here we specifically pick just the loader props we know we need to keep API exposure low.  Fine to add more
+    // here if we determine they're needed, but they should be picked explicitly (e.g. avoid "scope").
     public constructor(
-        props: ILoaderProps
+        props: Pick<ILoaderProps, "urlResolver" | "documentServiceFactory" | "codeLoader">
```

Comment:
Any thoughts on defining these strings as const's?

Reply:
Could do, or could have some IModelLoaderProps that is defined this way.  But I think having the explicit picks here probably makes for slightly more helpful intellisense?  My primary goal was just to match the call signature to the implementation rather than silently dropping configs on the ground.

I think some of this might end up changing anyway.  When I started looking at the other TODO (the one about the create new request) I realized the no-touch (parameterless) generateCreateNewRequest was a bit more challenging for ODSP.  So if we circumvent that issue via migrating the container in-place (so the Migrator doesn't need to call `attach()`) then the attach flow probably changes, which in turn probably changes the responsibility/construction of the ModelLoader.